packages/cli/src/util/__tests__/paths.spec.ts

```diff
@@ -75,7 +75,7 @@ describe('createExamplePath()', () => {
           },
           responses: [{ code: '200' }],
         }),
-        r => expect(r).toEqual('/path?p[a][aa]=1&p[a][ab]=2'),
+        r => expect(r).toEqual('/path?p%5Ba%5D%5Baa%5D=1&p%5Ba%5D%5Bab%5D=2'),
```

Comment:
This an similar lines need comment: the library is performing decodeURIComponent on passed arguments. That actually fixes a "bug". I used quotes because both versions put into the browser will work. The current one is more correct.

Reply:
Hmm, I am afraid I do not understand what's going on here — can you elaborare a little bit more?

Reply:
Sure. Previous library (`uri-templates`) did not care about whether input params are already uri encoded or not. That's why chars like '[' or ']' were visible in final urls. The new lib - `uri-template-lite` does encodeUriComponent on each params - that's the source of difference.

Both forms are valid though if you put it to the browser (browsers automatically encode some special characters) but I believe it is better that we provide already encoded form. So no relaying on browsers, our example uris will be ready to use.